File: Beginner/1009 (Salary with Bonus)/1009.java
import java.io.IOException;
import java.util.Scanner;
import java.text.DecimalFormat;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);
        DecimalFormat df = new DecimalFormat("0.00");
        String nome = ler.next();
        double salario = ler.nextDouble();
        double venda = ler.nextDouble();
        
        
        double result = (salario + (venda * 0.15));
        System.out.println("TOTAL = R$ " + df.format(result));
        
    }
 
}

File: Beginner/1009 (Salary with Bonus)/1009.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            string nome = Console.ReadLine();
            double salario = double.Parse(Console.ReadLine());
            double venda = double.Parse(Console.ReadLine());

            Console.WriteLine("TOTAL = R$ " + string.Format("{0:0.00}", venda * 0.15 + salario));


        }
    }
}